src/GitHub.VisualStudio/Services/UsageTracker.cs

```diff
@@ -194,21 +204,23 @@ void TimerTick(object sender, EventArgs e)
 
         async Task TimerTick()
         {
-            Debug.Assert(client != null, "TimerTick should not be triggered when there is no IMetricsService");
+            Initialize();
 
-            // Subsequent timer ticks should occur every 8 hours.
-            timer.Interval = TimeSpan.FromHours(8);
+            if (firstRun)
+            {
+                IncrementLaunchCount();
+                timer.Interval = TimeSpan.FromHours(8);
+                firstRun = false;
+            }
 
             if (userSettings == null)
```

Comment:
Because this might get called from a background thread (if it was triggered from a background thread) and we were doing `GetExportedValue`, which needs to be called on the main thread. Speaking of which, we need to be very careful about what we call in `Initialize`, there's side effects that can deadlock, I'll comment there.